### ISSUE TITLE
Can't get __dirname to work

### ISSUE THREAD

**Issue by schadenn** [calvinmetcalf/rollup-plugin-node-globals#23]
Hi,

I'm trying to use the plugin to use `__dirname` in my component src somewhere. I already dug quite deep into your code and found that the plugin looks for `__dirname` in my code and if it's there it puts a `import __dirname from 'node-globals:dirname'` at the top of the file.

Just how is this `import from 'node-globals:dirname` supposed to work (maybe if you explain, I can find the solution)?
There is no npm module called node-globals being installed.
So everytime I'm trying to use __dirname rollup just fails with:
```
"Could not load  node-globals:dirname (imported by
           [...]/DevPage.js): The argument 'path' must be a string
           or Uint8Array without null bytes. Received '\u0000node-globals:dirname'"
```
Also not quite sure why there is a `\0` character in front of the `DIRNAME` path in your code.

Thanks in advance.

**Comment by ca-miked** [comment]
I'm having the same exact issue

**Comment by kuraga** [comment]
Yes, strange, null byte (`\0`) is really in [these lines](https://github.com/calvinmetcalf/rollup-plugin-node-globals/blob/v1.4.0/src/index.js#L10-L11). But why?

And why exactly `'\0node-globals:dirname'`/`\0'node-globals:filename'`?What does it mean?

/cc @calvinmetcalf

**Comment by kuraga** [comment]
Null byte (`\0`) is really in [these lines](https://github.com/calvinmetcalf/rollup-plugin-node-globals/blob/v1.4.0/src/index.js#L10-L11). It was introduced in #3 .

**Comment by kuraga** [comment]
@calvinmetcalf , ping

**Comment by schadenn** [comment]
@kuraga See: https://github.com/rollup/rollup/issues/2881
It doesn't polyfill `__dirname` though currently.

I'm not sure what your usecase is. Mine was to do some node fs operations during runtime which turned out to not being a very good idea in the first place. If you've got a valid use-case maybe raise an issue over at [rollup-plugin-node-polyfills](https://github.com/ionic-team/rollup-plugin-node-polyfills).

**Comment by kuraga** [comment]
@schadenn , thanks! My usecase is [using `_dirname` by a (development) dependency of my project](https://github.com/thunks/tman/blob/925216e0084d1d4cce3595cd611c385ab6e17e05/lib/tman.js#L65).

**Comment by kuraga** [comment]
@schadenn , thanks!

It [supports](https://github.com/ionic-team/rollup-plugin-node-polyfills/commit/9fb7678e28b23b7efcf9ec1a7054f0c6a88676b2), doesn't it?

My usecase is [using `_dirname` by a (development) dependency of my project](https://github.com/thunks/tman/blob/925216e0084d1d4cce3595cd611c385ab6e17e05/lib/tman.js#L65).

**Comment by schadenn** [comment]
@kuraga Oh okay, yeah might be that it supports it. I didn't see it being listed in the Readme and did no further investigation.
In that case I wish you good luck trying that plugin :)
